Vienna Tests/models/ArticleTests.m

```diff
@@ -348,4 +349,18 @@ - (void)testExpandArticleEnclosureFileName
     XCTAssertEqualObjects(expandedString, expectedString);
 }
 
+- (void)testURLIDNinArticleView
+{
+    ArticleView * view = [[ArticleView alloc] initWithFrame:NSMakeRect(0, 0, 100, 100)];
```

Comment:
I'm not happy with having the view part of a test suite that is meant to be specifically about the model. Is there a different way we can do this?

Reply:
To be honest, this is not due to the test using the view, it is due to the view doing controller / model transformation tasks. In my effort to convert the article view to a WebKit-based WebView, this was the first thing I factored out into a different object. As soon as this is done, we can also convert the test such that it only uses model / controller classes.

Reply:
agreed, let's be pragmatic for the time being…